Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Group of VMs/Performance Analysis for a Group of VMs.workbook

```diff
@@ -242,12 +242,34 @@
             "isHiddenWhenLocked": true,
             "timeContextFromParameter": null,
             "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "ce228deb-88eb-4438-9dce-6c6d1972ff09",
+            "version": "KqlParameterItem/1.0",
+            "name": "IsNetworkCounter",
+            "type": 1,
+            "isRequired": false,
+            "query": "let counter = dynamic({Counter});\r\nprint tostring(counter.object == 'Network Adapter')",
+            "isHiddenWhenLocked": true,
+            "timeContextFromParameter": null,
+            "resourceType": "microsoft.operationalinsights/workspaces"
           }
         ],
         "resourceType": "microsoft.operationalinsights/workspaces"
       },
       "conditionalVisibility": null
     },
+    {
+      "type": 1,
+      "content": {
+        "json": "#### ℹ Additional counters are being utilized to display the data shown below"
```

Comment:
There are limitations to passing a custom query since we cannot dereference parameters inside of a string. This will limit certain options in the workbooks, such as selecting a custom aggregation, timerange, etc. The workbooks team is aware of this and have it in their backlog. Given these limitations option 2 is the ideal solution.

I'll update the disclaimer to include additional information how the additional counters are used. Thanks.

Reply:
Actually, I am looking at #92 and the second approach used in this PR tracks closely to the "query snippets" that you are using, although your method is more flexible. I think I can eventually boil this down to a single chart using your method. Let me know what you think.

Reply:
That sounds great. Even I really like the second approach.